### PR TITLE
DOMA-1564 Ticket deadline

### PR DESCRIPTION
**Pull request by nomerdvadcatpyat**
- Added a `deadline` field to the `Ticket` schema. By default, the deadline occurs 2 days after the `ticket` is created.
- Changed the layout in `BaseTicketForm` in accordance with layouts
- Made `tickets` sorting based on `deadline`. Now the `tickets` with the status "Open" are pinned on top. Next are expired `tickets`. Next come `tickets` that have less than 24 hours left to complete. All other `tickets` follow.
- Fixed `DatePicker` localization by adding `AntdDayjsWebpackPlugin`

---
- [x] make support for `date` in `TicketChange`
- [x] Update layout in `CreateTicketForm` in `callcenter`

### CODE REVIEW

File: apps/condo/next.config.js [hunk 1]
```diff
@@ -44,4 +45,11 @@ module.exports = withTM(withLess(withCSS({
             },
         ]
     },
+    webpack: (config) => {
+        const plugins = config.plugins
+
+        config.plugins = [ ...plugins, new AntdDayjsWebpackPlugin() ]
```
Comment (pahaz) [inline apps/condo/next.config.js:51]: Please add `NOTE` what it is and why we need it?

Comment (pahaz) [inline apps/condo/next.config.js:51]:
```suggestion
        // NOTE: Replace Moment.js with Day.js in antd project
        config.plugins = [ ...plugins, new AntdDayjsWebpackPlugin() ]
```

File: apps/condo/domains/ticket/constants/statusTransitions.js [hunk 2]
```diff
@@ -28,6 +28,10 @@ const COUNTRY_RELATED_STATUS_TRANSITIONS = {
 
 const TICKET_ORDER_BY_STATUS = {
     [STATUS_IDS.OPEN]: 100,
+    [STATUS_IDS.IN_PROGRESS]: 200,
```
Comment (pahaz) [inline apps/condo/domains/ticket/constants/statusTransitions.js:31]: we can create our own status ids for organizations. You need to pay attantion

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/ticket/constants/statusTransitions.js (hunk 2 above)
Comment (nomerdvadcatpyat) [reply apps/condo/domains/ticket/constants/statusTransitions.js:31]: In this case, I think I will make `order = 100` only for the "Open" status, for all others set 200 (not only for default ones)

**Review by sitozzz — CHANGES_REQUESTED** [review]
Need to recreate migration file, because at origin we already have `0094`

**Review by sitozzz — APPROVED** [review]

**Pull request merged**